invokeai/app/services/model_install/model_install_default.py

```diff
@@ -783,8 +783,9 @@ def _multifile_download(
         # So what we do is to synthesize a folder named "sdxl-turbo_vae" here.
         if subfolder:
             top = Path(remote_files[0].path.parts[0])  # e.g. "sdxl-turbo/"
-            path_to_remove = top / subfolder.parts[-1]  # sdxl-turbo/vae/
-            path_to_add = Path(f"{top}_{subfolder}")
+            path_to_remove = top / subfolder  # sdxl-turbo/vae/
+            subfolder_rename = subfolder.name.replace("/", "_").replace("\\", "_")
+            path_to_add = Path(f"{top}_{subfolder_rename}")
```

Comment:
I took a closer look at this. Logic seems ok to me, but this line seems unnecessary:

`subfolder.name.replace("/", "_").replace("\\", "_")`

`subfolder.name` is the final path component, so shouldn't contain any path separators. Am I missing something here? Do we want to be keeping the full path here rather than only the last component? (concern with that is that it would change the file name when we are downloading a single file)